### DISCUSSION TITLE
Is there a documentation for the sources?

### DISCUSSION THREAD

**Discussion by AbdulazizAljumaia** [Shadowss/TravianZ#60] [Q&A]
Greetings, 
thank you so much for you amazing work. 

I just want to know if there is a documentation for the work as I found it very dificult to modify it. I am trying to amend the CSS files or to comeplety change the style of the layout. Thank you.

**Comment by Shadowss** [comment]
No, unfortunately there is no documentation. There are more than 50 programers that write this code
